src/components/Other/Team.vue

```diff
@@ -67,12 +82,48 @@ export default {
     title: 'Team'
   },
   computed: {
-    ...mapGetters('team', ['team'])
+    ...mapGetters('team', ['members'])
+  },
+  methods: {
+    iconUrl: function(type) {
+      switch (type) {
+        case 'facebook':
+          return 'https://cdn3.iconfinder.com/data/icons/free-social-icons/67/facebook_circle_color-512.png'
+          break
+        case 'email':
+          return 'https://cdn2.iconfinder.com/data/icons/jetflat-multimedia/90/004_009_mail_email_envelope_message-512.png'
+          break
+        case 'g-plus':
+          return 'https://cdn0.iconfinder.com/data/icons/social-media-2091/100/social-03-512.png'
+          break
+        case 'linkedin':
+          return 'https://cdn4.iconfinder.com/data/icons/miu-flat-social/60/linkedin-512.png'
+          break
+        case 'mobile':
+          return 'https://cdn3.iconfinder.com/data/icons/mobile-functions/154/call-512.png'
+          break
+        case 'twitter':
+          return 'https://cdn4.iconfinder.com/data/icons/social-media-icons-the-circle-set/48/twitter_circle-512.png'
+          break
+      }
+    },
+    openLink: function(link, type) {
+      if (
+        type == 'facebook' ||
+        type == 'linkedin' ||
+        type == 'twitter' ||
+        type == 'g-plus'
+      ) {
+        window.location.href = link
+      } else {
+        window.alert(link)
+      }
```

Comment:
Instead of this method, use anchor tag. `<a href="url"></a>`

Reply:
Anchor tag isn't working with <v-img>

Reply:
Wrap the v-btn in anchor